tests/span_ext_tests.cpp

```diff
@@ -193,10 +194,18 @@ TEST(span_ext_test, make_span_from_container_constructor)
 
 TEST(span_test, interop_with_gsl_at)
 {
+    const auto terminateHandler = std::set_terminate([] {
+        std::cerr << "Expected Death. interop_with_gsl_at";
+        std::abort();
+    });
+    const auto expected = GetExpectedDeathString(terminateHandler);
+
     int arr[5] = {1, 2, 3, 4, 5};
     gsl::span<int> s{arr};
     EXPECT_TRUE(at(s, 0) == 1);
     EXPECT_TRUE(at(s, 1) == 2);
+    EXPECT_DEATH(at(s, 5), expected);
+    EXPECT_DEATH(at(s, 6), expected);

```

Comment:
adding a `-1` index test-case would add coverage to the `>= 0` condition

Reply:
I saw there `span_tests` `interop_with_gsl_at` and `at_tests` `std_span` differ. I think they should just do the same thing except one using `std::span` and the other using `gsl::span`.